packages/framework-core/src/services/graphql/graphql-type-informer.ts

```diff
@@ -107,4 +141,13 @@ export class GraphQLTypeInformer {
     }
     return inputFields
   }
+
+  private generateOperationEnumValuesFor(operationsEnum: AnyClass): GraphQLEnumValueConfigMap {
+    const enumValuesConfig: GraphQLEnumValueConfigMap = {}
+    for (const opSymbol in operationsEnum) {
+      const opName = (operationsEnum as any)[opSymbol]
```

Comment:
`any` police 👮🏻‍♂️

What's the contents of `operationsEnum`? Maybe we can type it in a better way? Can you explain more please?

Reply:
I didn't create that function, it was added after a rebase, so there isn't much I can tell you here 😅

Reply:
Ok, keep circulating 👮